build.go

```diff
@@ -32,13 +41,31 @@ func Build(buildProcess BuildProcess, logs scribe.Emitter) packit.BuildFunc {
 			return packit.BuildResult{}, err
 		}
 
+		modCacheLayer.Launch = true
```

Comment:
@TisVictress What was the motivation behind making the modules cache layer available at launch time?

Reply:
I did some more investigating and realized that it's currently not possible to access the SBOMs for layers that aren't marked for launch. See [this conversation on a lifecycle issue implementing SBOM support](https://github.com/buildpacks/lifecycle/issues/735#issuecomment-1015849216). That means that currently, there's no way for an end user to access the SBOM generated by this buildpack unless the layer is marked for launch. (I'm guessing you already know this and it's why you marked the layer for launch?)

I don't think we should change this buildpack to mark the modules layer for launch – for security/hardening and optimization reasons we don't want the modules layer present in the app image. But unfortunately I think that means there's no way to test or use this feature until the lifecycle and/or pack makes the build SBOM available 😞 .

Reply:
@fg-j I need to be able to test that the BOM files exist in the integration test.

Reply:
So, I pinged CNB folks about this, and it looks like there's now an PR to enable grabbing build-time BOMs: https://github.com/buildpacks/pack/pull/1359

Since this work is blocked anyway, let's wait to see if the build-BOM feature lands soon. That way we can integration test without changing the API of the buildpack.